# awesome-genome-visualization

A list of interesting genome visualizers, genome browsers, or
genome-browser-like implementations

## Genome browsers

### Desktop

- [BasePlayer](https://baseplayer.fi/)
- [IGB](https://bioviz.org/)
- [IGV](https://igv.org/) (java)
- [Savant](http://bioinformatics-ca.github.io/savant_genome_browser_lab_2015/)
- [Tablet](https://ics.hutton.ac.uk/tablet/)

### Web-based

- [Biodalliance](http://www.biodalliance.org/)
- [Ensembl genome browser](https://useast.ensembl.org/Homo_sapiens/Location/View?r=17:63992802-64038237)
- [Ensembl genome browser 2020 edition](http://2020.ensembl.org/)
- [GBrowse+GBrowse 2](http://gmod.org/wiki/GBrowse)
- [GenomeMaps](http://www.genomemaps.org/) (used on ICGC)
- [HiGlass](https://higlass.io) (has epigenomics focus but quite advanced)
- [IGV.js](https://github.com/igvteam/igv.js/)
- [JBrowse](http://jbrowse.org/) [can run on desktop via Electron]
- [JBrowse 2](http://jbrowse.org/jb2) (under development)
- [Kero-BROWSE](https://github.com/DBKERO/genome_browser) ([example](https://kero.hgc.jp/examples/CLCL/hg38/index.html))
- [NCBI Genome Data Viewer](https://www.ncbi.nlm.nih.gov/genome/gdv/)
- [Nucleome browser](https://vis.nucleome.org/entry/)
- [Pileup.js](https://github.com/hammerlab/pileup.js/)
- [UCSC genome browser](https://genome.ucsc.edu/)
- [Trackster](https://galaxyproject.org/learn/visualization/) (in Galaxy)
- [WashU epigenomics browser](https://epigenomegateway.wustl.edu/) [source](https://github.com/lidaof/eg-react)
- [Valis browser](https://valis.bio/) - used on encodeproject.org (uses webGL)
- [Zenbu](http://fantom.gsc.riken.jp/zenbu/)

### Commercial

- [10x genomics](https://support.10xgenomics.com/genome-exome/software/visualization/latest/structural-variants)
- [Alamut](https://www.interactive-biosoftware.com/alamut-visual/)
- [Benchling](https://www.benchling.com/)
- [ATCC Genome Portal](https://genomes.atcc.org)
- [CLC Genomics workbench](https://digitalinsights.qiagen.com/news/blog/discovery/structural-variant-detection-using-clc-genomics-workbench/)
- [DNASTAR](https://www.dnastar.com/)
- [Geneious](https://www.geneious.com/)
- [Genestack](https://genestack.com/blog/2015/05/28/navigation-in-genestack-genome-browser/)
- [Golden Helix](http://goldenhelix.com/GenomeBrowse/)
- [Lucid viewer](https://lucidalign.com/)
- [MacVector](https://macvector.com/)
- [Persephone](https://persephonesoft.com/)
- [SnapGene](https://www.snapgene.com/)
- [Strand NGS](https://www.strand-ngs.com/)

### Hi-C and epigenomics focus

- [Chip Monk](http://www.bioinformatics.babraham.ac.uk/projects/chipmonk/)
- [Chipster](https://chipster.csc.fi/)
- [cisGenome Browser](http://www-personal.umich.edu/~jianghui/browser/screenshots.html)
- [Epilogos](http://compbio.mit.edu/epilogos/)
- [Epiviz](https://www.biorxiv.org/content/10.1101/865295v1.full.pdf)
- [HiGlass](https://higlass.io) (has epigenomics focus but quite advanced)
- [HiC browser](http://promoter.bx.psu.edu/hi-c/view.php) - UCSC-based
- [Juicebox](https://github.com/aidenlab/Juicebox) and [Juicebox.js](http://www.igv.org/doc/juiceboxjs.html)

## Special purpose tools

- [ApE](https://jorgensen.biology.utah.edu/wayned/ape/) (a plasmid editor)
- [Apollo](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC151184/) (old desktop version)
- [Apollo aka WebApollo](https://genomearchitect.readthedocs.io/en/latest/)
- [Chromatic](https://chromatic.nci.nih.gov/) (pub https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5987889/)
- [cnvCurator](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0766-y)
- [cnvkit](https://cnvkit.readthedocs.io/en/stable/plots.html)
- [CNVnator](http://www.abyzovlab.org/cnvnatorapplet)
- [CNVpytor](https://github.com/abyzovlab/CNVpytor)
- [DNASkittle](https://www.dnaskittle.com/) also see [FluentDNA](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7203487/)
- [EaSeq](https://easeq.net/screenshots/)
- [Gambit](http://labsergen.langebio.cinvestav.mx/bioinformatics/jacob/?p=473)
- [GenomeRibbon](http://genomeribbon.com/)
- [GenomeSpy](https://genomespy.app/examples/TCGA/BRCA/) [another demo](https://genomespy.app/examples/GWAS/OCAC-HGSOC/)
- [Genoverse](https://github.com/wtsi-web/Genoverse)
- [Gnomad browser](https://github.com/macarthur-lab/gnomadjs)
- [Gremlin](http://compbio-research.cs.brown.edu/software/Gremlin/GremlinOverview.pdf)
- [GTEX Locus browser](https://gtexportal.org/home/locusBrowserPage/ACTN3)
- [GWAS catalog browser](https://www.ebi.ac.uk/gwas/variants/rs1558902) at EBI
- [Hawkeye](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2007-8-3-r34)
- [Human genome dating](https://human.genome.dating/region/chr1_13250000) (made with vega/d3)
- [Icarus](http://quast.sourceforge.net/icarus.html)
- [Introgression browser](https://onlinelibrary.wiley.com/doi/pdf/10.1111/tpj.12800)
- [InGAP-SV](http://ingap.sourceforge.net/)
- [IslandPlot](https://bl.ocks.org/lairdm/c6c235dbfa6e6ee61565) (D3, SVG) ([source](https://github.com/lairdm/islandplot))
- [Locuszoom](http://locuszoom.org/)
- [LookSeq](https://genome.cshlp.org/content/early/2009/08/13/gr.093443.109) ([screenshot](https://www.sanger.ac.uk/tool/lookseq/)) ([github](https://github.com/magnusmanske/lookseq))
- [MagicViewer](http://bioinformatics.zj.cn/magicviewer/)
- [MalariaGEN](https://www.malariagen.net/apps/pf-crosses/1.0/#genome)
- [MapView](http://seqanswers.com/forums/showthread.php?t=1024) (note: links in original paper are dead. download link here https://sites.google.com/site/wjwdavy/)
- [NGB](https://github.com/epam/NGB)
- [Panoptes](https://github.com/cggh/panoptes) see [malariagen browser](https://www.malariagen.net/apps/pf3k/release_3/index.html#genomebrowser)
- [Parasight](https://baileylab.brown.edu/parasight/galframe.html)
- [RNASeqBrowser](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1346-2) UCSC-based
- [SEQing](https://github.com/malewins/SEQing)
- [SFARI](https://gene.sfari.org/database/human-gene/)
- [SNPViz](http://www.yeastrc.org/snipviz/4.HTML_Config_retrieve_newick_and_fasta_from_server/snip_viz_HTML_config_with_newick_clustering.html)
- [SNPitty](https://bitbucket.org/ccbc/snpitty/src/master/) (pub https://jmd.amjpathol.org/article/S1525-1578(17)30166-6/fulltext)
- [Staden](https://sourceforge.net/projects/staden/)
- [SplitThreader](http://splitthreader.com/) also see [GenomeRibbon](http://genomeribbon.com/)
- [TASUKE](https://ricegenome.dna.affrc.go.jp/) - from RAP-DB
- [UGENE](http://ugene.net/)
- [Varsome](<https://varsome.com/security-validation/?next=/variant/hg19/NM_000088.3(COL1A1):c.658C%3ET#community-contributions>)
- [Vials](http://vials.io/info/)
- [wasm bigwig demo browser](https://shk656461.github.io/index.html)

## Protein focused

- [ProteinPaint](https://pecan.stjude.cloud/proteinpaint)
- [Protael](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5963358/)
- [pViz](https://github.com/Genentech/pviz)

## Spreadsheet

- [BrowseVCF](https://github.com/BSGOxford/BrowseVCF) (pub https://pubmed.ncbi.nlm.nih.gov/27373737/)
- [CuteVCF](https://github.com/labsquare/CuteVCF)

## Exotic

- [BigTop](https://github.com/dnanexus/bigtop) ([virtual reality demo](https://blog.dnanexus.com/2019-05-21-bigtop-data-visualization/))
- [Cylindrical viewer](https://sourceforge.net/projects/cyl-viewer/) ([source](https://sourceforge.net/projects/cylindrical-alignment-app/))

## Heatmaps

- [UCSC Xena](https://xena.ucsc.edu/)
- [VIVA](https://www.biorxiv.org/content/10.1101/589879v1.full.pdf)

## Genome map, or comparative viewers

- [Aequatus](https://github.com/TGAC/Aequatus)
- [AliTV](https://alitvteam.github.io/AliTV/d3/AliTV.html)
- [Artemis comparison tool (ACT)](https://www.sanger.ac.uk/science/tools/artemis-comparison-tool-act)
- [Biodalliance comparative demo](http://biodalliance.org/dev/test-comparative.html)
- [Chromatiblock](https://www.biorxiv.org/content/10.1101/800920v2.full.pdf)
- [CMap](http://gmod.org/wiki/CMap)
- [CoGe](https://genomevolution.org/coge/)
- [Comparative assembly hub / "snake track"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4296145/) (UCSC based)
- [CVit](https://sourceforge.net/projects/cvit/)
- [Cvit.js](https://github.com/LegumeFederation/cvitjs)
- [CrossBrowse](https://github.com/shenkers/CrossBrowse) [pub](https://www.biorxiv.org/content/10.1101/272880v1.full)
- [EasyFig](https://mjsull.github.io/Easyfig/)
- [Edinburgh-Genome-Foundry/DnaFeaturesViewer](https://github.com/Edinburgh-Genome-Foundry/DnaFeaturesViewer)
- [Genomicus](https://www.genomicus.biologie.ens.fr/genomicus/)
- [genoplotR](http://genoplotr.r-forge.r-project.org/)
- [genome-plots-processing](https://github.com/filip-husnik/genome-plots-processing)
- [gggenomes](https://github.com/thackl/gggenomes)
- [Genome Context Viewer](https://www.legumefederation.org/gcv/phytozome_10_2/) ([blog post](https://www.legumefederation.org/en/blog/2019/06/25/genome-context-viewer-gcv-test-drive/))
- [GIVE](https://zhong-lab-ucsd.github.io/GIVE_homepage/)
- [GBrowse_syn](http://gmod.org/wiki/GBrowse_syn)
- [mauve-viewer](https://github.com/nconrad/mauve-viewer)
- [MCScan (python version)](<https://github.com/tanghaibao/jcvi/wiki/MCscan-(Python-version)>)
- [MCScanX](http://chibba.pgml.uga.edu/mcscan2/)
- [MizBee](https://ieeexplore.ieee.org/document/5290692)
- [Multiple genome viewer](http://www.informatics.jax.org/mgv/) (jax)
- [mGSV](https://github.com/kashmatic/mGSV)
- [OMA local synteny browser](https://omabrowser.org/oma/synteny/ADH1A_HUMAN/)
- [pafr](https://github.com/dwinter/pafr)
- [plotsr](https://schneebergerlab.github.io/syri/plotsr.html) from SyRI
- [progressiveMauve viewer](http://darlinglab.org/mauve/user-guide/viewer.html)
- [Synteny browser](https://github.com/TheJacksonLaboratory/syntenybrowser) (jax)
- [SynTView](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3849071/) ([webpage](http://hub18.hosting.pasteur.fr/SynTView/documentation/Synteny/))
- [SyMap](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105427/)
- [Synima](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5697234/)
- [SynVisio](https://synvisio.github.io/#/)
- [VISTA browser](http://pipeline.lbl.gov/cgi-bin/gateway2)
- [XMatchView](https://www.bcgsc.ca/resources/software/xmatchview)

## Multiple sequence alignment viewer/Logo

- [AA](http://www.bioinformatics.org/strap/aa) also see Java web applet version [STRAP](http://www.bioinformatics.org/strap/)
- [abrowse](https://github.com/ihh/abrowse)
- [alignment.js](https://github.com/veg/alignment.js)
- [AlignmentViewer](https://alignmentviewer.org/)
- [AliView](http://www.ormbunkar.se/aliview/)
- [BioJS/MSA](https://msa.biojs.net/) also see [msa/R](https://github.com/zachcp/msaR)
- [Boxshade](https://embnet.vital-it.ch/software/BOX_form.html)
- [ESPript](http://espript.ibcp.fr/ESPript/ESPript/)
- [Jalview](https://www.jalview.org/)
- [JSAV](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4304231/) [link](http://www.bioinf.org.uk/software/jsav)
- [LogoJS](https://logojs.wenglab.org/app/gallery/)
- [MView](https://www.ebi.ac.uk/Tools/msa/mview/)
- [React MSAViewer](https://github.com/plotly/react-msa-viewer)
- [seqotron](https://github.com/4ment/seqotron)
- [TeXShade](http://mirrors.ibiblio.org/CTAN/macros/latex/contrib/texshade/texshade.pdf)
- [Wasabi](http://wasabiapp.org/)

## Static visualizations

- [aCNVViewer](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0189334) ([github](https://github.com/FJD-CEPH/aCNViewer))
- [BAMScale](https://github.com/ncbi/BAMscale)
- [ChIA-Pipe](https://www.biorxiv.org/content/10.1101/506683v1.full.pdf)
- [Chimeraviz](https://github.com/stianlagstad/chimeraviz)
- [ClustersPloter](https://github.com/orangeSi/ClustersPloter)
- [CNView](https://github.com/RCollins13/CNView)
- [CNVpytor](https://github.com/abyzovlab/CNVpytor)
- [CNVPlot](https://github.com/dantaki/CNVplot)
- [DNAPlotLib](https://github.com/VoigtLab/dnaplotlib)
- [ExonIntron](http://wormweb.org/exonintron) from WormBase
- [FeatureViewer](https://github.com/calipho-sib/feature-viewer)
- [ggbio](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2012-13-8-r77)
- [GGgenes](https://github.com/wilkox/gggenes)
- [GGsashimi](https://github.com/guigolab/ggsashimi)
- [gtrellis](http://bioconductor.org/packages/devel/bioc/vignettes/gtrellis/inst/doc/gtrellis.html)
- [genoPlotR](http://genoplotr.r-forge.r-project.org/)
- [Geneviz](https://jrderuiter.github.io/geneviz/usage.html)
- [Genome STRiP](http://software.broadinstitute.org/software/genomestrip/)
- [GenomeTools](http://genometools.org/)
- [GenomeView](https://github.com/nspies/genomeview)
- [GSDraw](https://wheat.pw.usda.gov/piece/GSDraw.php)
- [Gviz](https://bioconductor.org/packages/release/bioc/html/Gviz.html)
- [Hagfish](https://github.com/mfiers/hagfish/wiki/Plots)
- [HiCPlotter](https://github.com/kcakdemir/HiCPlotter)
- [JVarKit](http://lindenb.github.io/jvarkit/) examples http://lindenb.github.io/jvarkit/BamToSVG.html
- [Lollipops](https://github.com/joiningdata/lollipops)
- [Mason](http://www.yeastrc.org/mason/)
- [Millefy](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7053140/) single cell
- [Pairoscope](http://pairoscope.sourceforge.net/)
- [pauvre](https://github.com/conchoecia/pauvre) (see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6991124/figure/fig-3/ for example)
- [PureScript genetics browser](https://github.com/chfi/purescript-genetics-browser)
- [Methplotlib](https://www.biorxiv.org/content/10.1101/826107v1.full.pdf)
- [RACER](https://oliviasabik.github.io/RACERweb/articles/IntroToRACER.html)
- [RepViz](https://bmcresnotes.biomedcentral.com/articles/10.1186/s13104-019-4473-z)
- [Samplot](https://github.com/ryanlayer/samplot)
- [SashimiPlot](https://miso.readthedocs.io/en/fastmiso/sashimi.html)
- [shebam](https://github.com/dlrice/shabam)
- [shinyChromosome](http://150.109.59.144:3838/shinyChromosome/)
- [SNPduo](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0006711)
- [SparK](https://www.biorxiv.org/content/10.1101/845529v1.full.pdf)
- [Spliceclust](https://github.com/pkimes/spliceclust)
- [Sushi](https://www.bioconductor.org/packages/release/bioc/vignettes/Sushi/inst/doc/Sushi.pdf) (R package)
- [pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks)
- [svist4get](https://link.springer.com/article/10.1186/s12859-019-2706-8)
- [svviz2](https://github.com/nspies/svviz2)
- [svviz](https://github.com/svviz/svviz)
- [svv](https://github.com/ryanlayer/svv) (stupid simple structural variant view)
- [TnT](https://tnt.marlin.pub/articles/introduction)
- [trackViewer](https://bioconductor.org/packages/release/bioc/vignettes/trackViewer/inst/doc/trackViewer.html) (R)

## Circular

- [Anvio](https://peerj.com/articles/1319/)
- [BioCircos.js](http://bioinfo.ibp.ac.cn/biocircos/)
- [Circos](http://circos.ca/)
- [CircosJS](https://github.com/nicgirault/circosJS)
- [Circleator](http://jonathancrabtree.github.io/Circleator/index.html)
- [circlize](https://github.com/jokergoo/circlize)
- [DNAPlotter](https://www.sanger.ac.uk/science/tools/dnaplotter)
- [GenomeProjector](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2636772/) [multiple uses, not just circular]
- [JupiterPlot](https://github.com/JustinChu/JupiterPlot)
- [mummer2circos](https://github.com/metagenlab/mummer2circos)
- [OGDRAW](https://www.biorxiv.org/content/10.1101/545509v1.full.pdf)

## Dotplot

- [cgplot](https://github.com/dfguan/cgplot)
- [D-GENIES](https://github.com/genotoul-bioinfo/dgenies)
- [Delly-maze](https://github.com/dellytools/maze)
- [Discoplot](https://github.com/mjsull/DiscoPlot)
- [Dot](https://github.com/dnanexus/dot)
- [Dotlet](https://dotlet.vital-it.ch/)
- [Dotplot (iLambda)](https://github.com/iLambda/dotplot)
- [Dotplot (chirimoyo)](https://chirimoyo.ac.uma.es/bitlab/portfolio/dotplot/)
- [dotPlotly](https://github.com/tpoorten/dotPlotly/)
- [Gepard](http://cube.univie.ac.at/gepard)
- [Jdot](https://github.com/LyonsLab/jdot)
- [LAST](http://last.cbrc.jp/) (package contains a dotplotter)
- [mcutils](https://github.com/mchaisso/mcutils) [source](https://twitter.com/mjpchaisson/status/1040363992206569472)
- [Minidot](https://github.com/thackl/minidot)
- [Mummer](http://mummer.sourceforge.net/) (contains a GNUplot script, mummerplot, for dotplotting)
- [mummer-idotplot](https://github.com/ryought/mummer-idotplot)
- [Redottable](https://github.com/s-andrews/redotable)
- [syntenyPlotByR](https://github.com/shingocat/syntenyPlotByR/blob/master/out.png)

## Karyotype

- [chromoMap](https://www.biorxiv.org/content/10.1101/605600v1.full.pdf)
- [Ideogram](https://github.com/RCollins13/HumanIdiogramLibrary) HumanIdeogramLibrary
- [NCBI Genome Decoration](https://www.ncbi.nlm.nih.gov/genome/tools/gdp/)
- [eweitz/Ideogram](https://github.com/eweitz/ideogram)
- [KaryotypeSVG](https://github.com/andreasprlic/karyotypeSVG)
- [rnaseqview](https://github.com/NCBI-Hackathons/rnaseqview)
- [karyoploteR](https://github.com/bernatgel/karyoploteR) (R package)
- [UCSC Genome Graphs](http://genome.ucsc.edu/cgi-bin/hgGenome)
- [Ideoplot](https://github.com/mchaisso/Ideoplot)

## Command line

- [ASCIIGenome](https://github.com/dariober/ASCIIGenome)
- [Alan](https://github.com/mpdunne/alan)
- [Alv](https://github.com/arvestad/alv)
- [Bamcov](https://github.com/fbreitwieser/bamcov)
- [bioSyntax](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2315-y)
- [BLAST+](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/) <-- reading -outfmt 0 pairwise output is essentially a textual/command line bioinformatics tool
- [Hapviz](https://github.com/ekg/hapviz)
- [plotReads](http://campuspress.yale.edu/knightlab/ruddle/plotreads/) [includes static graphic and text visualization]
- [pyBamView](https://mgymrek.github.io/pybamview/)
- [Rna Alignment Viewers](http://biowiki.org/wiki/index.php?title=Rna_Alignment_Viewers&redirect=no) ref https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109877/
- [Sam2pairwise](https://github.com/mlafave/sam2pairwise)
- [Samtools tview](http://www.htslib.org/) ([keyboard shortcuts](img/tview.png))
- [SvABA](https://github.com/walaj/svaba#alignmentstxtgz) has a particular text visualization of breakpoints
- [VizAln](https://github.com/tfwillems/HipSTR#alignment-visualization) from HipSTR
- [Samtools depth visualization](https://twitter.com/yokofakun/status/1178686978541441025)

## Graph genomes

- [AGB assembly graph browser](https://almiheenko.github.io/AGB/index.html)
- [Bandage](https://github.com/rrwick/Bandage/)
- [gGnomes.js](https://github.com/mskilab/gGnome.js)
- [MoMi-G](https://github.com/MoMI-G/MoMI-G/)
- [sequence tube map](https://github.com/vgteam/sequenceTubeMap)
- [graphgenomeviewer](https://cmdcolin.github.io/graphgenomeviewer/) (note: I made this during bcc2020 hackathon)
- [odgi](https://github.com/vgteam/odgi) (ref https://pangenome.github.io/odgi/odgi_docs.html#_odgi_viz1)

## Pangenome

- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4547177/
- RPAN http://cgm.sjtu.edu.cn/3kricedb/visualization/?tracks=DNA%2Cgene%2CPF
- panX http://pangenome.tuebingen.mpg.de/
- Panache https://github.com/SingingMeerkat/Panache

## Mobile

- iGenomics - https://www.biorxiv.org/content/10.1101/2020.02.11.944132v1

# Other lists

- [Awesome Bioinformatics](https://github.com/danielecook/Awesome-Bioinformatics)
- [Genocat](http://genocat.tools/) large collection of genomic visualizations with great review paper accompaniment https://onlinelibrary.wiley.com/doi/full/10.1111/cgf.13727
- [awesome-biological-visualizations](https://github.com/keller-mark/awesome-biological-visualizations)

Send in PRs for more stuff!

## Note

Update: I have started to convert this into a CSV format for more rich
metadata. See [TOOLS.csv](TOOLS.csv). The formatting on this is not perfect and
if you are reading into R use `read.table('TOOLS.csv', fill=T)`. PRs
contributing to the formatting, or additional richness of this metadata are
welcome! I still like the "awesome-list" format but it is not structured enough
to contain extra columns of data, etc. Possibly the awesome-list will be
generated by the CSV in the future

## License

This README.md and TOOLS.csv are CC0
https://creativecommons.org/publicdomain/zero/1.0/
